.github/workflows/deploy.yml

```diff
@@ -52,11 +46,15 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-2
 
+      - name: Store ENV from AWS SecretManager
+        uses: say8425/aws-secrets-manager-actions@v2
+        with:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          SECRET_NAME: ${{ input.cognito_app_secret }}
+
```

Comment:
Sorry about the confusion in the slack messages, but I don't think we can use this here. The code does require the 'COGNITO_APP_SECRET' env variables. 

This can be used in cases where instead of storing the secrets in GitHub Secrets, we want to put them in AWS Secrets Manager and have the actions automatically export to env from there.

Reply:
I thought we were going to use GH secrets for the AWS key/ID + the cognito secret to populate environment variables here, no?

Reply:
Perhaps we can take this to slack again, as I'm afraid there are multiple senses of some words at play here and I'm missing something important